modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/Field/Field.es.js

```diff
@@ -148,6 +150,7 @@ const FieldLazy = ({
 			visible
 			{...field}
 			{...otherProps}
+			{...module.properties}
```

Comment:
Instead of having this new `properties` can't we just add editorConfig directly in the JSON?

Reply:
Yes, I was thinking of just reducing the hack a little so it doesn't look so exclusive to the `editorConfig` 🤪